.github/workflows/release.yml

```diff
@@ -190,7 +190,7 @@ jobs:
           SHORT_COMMIT: ${{ steps.vars.outputs.short_commit }}
           DATE: ${{ steps.vars.outputs.date }}
         run: |
-          ko build --bare --tags latest --tags ${{ env.TAG }}
+          ko build --bare --tags latest --tags ${{ env.TAG }} --platform=linux/amd64,linux/arm64
```

Comment:
nit: should we consolidate this configuration into the ko config file?

https://github.com/odigos-io/odigos/blob/main/cli/.ko.yaml